lib/octokit/client/licenses.rb

```diff
@@ -0,0 +1,50 @@
+module Octokit
+  class Client
+
+    # Methods for licenses API
+    #
+    module Licenses
+
+      LICENSES_PREVIEW_MEDIA_TYPE = "application/vnd.github.drax-preview+json".freeze
+
+      # List all licenses
+      #
+      # @see https://developer.github.com/v3/licenses/#list-all-licenses
+      # @return [Array<Sawyer::Resource>] A list of licenses
+      # @example
+      #   Octokit.licenses
+      def licenses(license_name = '', options = {})
```

Comment:
@pengwynn Check the update. There were only one method previously, and I decided to separate them but forget to delete that parameter....